pkg/containertools/imagereader.go

```diff
@@ -42,7 +42,12 @@ func NewImageReader(containerTool string, logger *logrus.Entry) ImageReader {
 	}
 }
 
-func (b ImageLayerReader) GetImageData(image, outputDir string) error {
+func (b ImageLayerReader) GetImageData(image, outputDir string, opts ...GetImageDataOption) error {
```

Comment:
I don't see anything (non-test) calling this with the new option - did I miss that somewhere?

Reply:
it's just to inject the temp dir for tests so that we can inject the result of the `docker save`